src/components/component-playground.jsx

```diff
@@ -115,6 +117,23 @@ module.exports = React.createClass({
       _.assign(params, this.state.fixtureUnserializableProps);
 
       return _.merge(params, _.omit(this.state.fixtureContents, 'state'));
+    },
+
+    splitPane: function() {
+      return {
+        component: SplitPane,
+        key: 'editorPreviewSplitPane',
+        split: this._getOrientationDirection(),
+        defaultSize: localStorageLib.get('splitPos'),
+        onChange: (size => {localStorageLib.set('splitPos', size)}),
```

Comment:
Please move the logic inside a method and call it here. If you'll write a method that receives `size` and call `localStorageLib` you can test it with ease.

`onChange: this.onChange` => `props.onChange.to.equal(component.onChange)`
`component.onChange('xxx')` => `expect(localStorageLibStub.to.have.been.calledWith('splitPos', 'xxx')`.